feat: add Collect Cards guide (#102)
Basis-Theory/developers.basistheory.com: src/components/docs/_configure-elements-section.mdx

import useBaseUrl from "@docusaurus/useBaseUrl";
import ThemedImage from "@theme/ThemedImage";

import { Card } from "@site/src/components/shared/Card"
import styles from "@site/src/pages/index.module.css";
import Android from "@site/static/img/sdk/logos/android.svg";
import JavaScript from "@site/static/img/sdk/logos/javascript.svg";
import ReactSvg from "@site/static/img/sdk/logos/react.svg";

<!-- This doesn't contain headings because they are not supported by the TOC -->
<!-- https://github.com/facebook/docusaurus/issues/3915 -->

> [Basis Theory Elements](/docs/concepts/elements) is available for the following technologies. Click below for detailed instructions on how to install and configure it.

Reviewer: ```suggestion
[Basis Theory Elements](/docs/concepts/elements) are available on the following technologies. Click below for detailed instructions on how to install and configure them.
```
